tutorials/workflows/basics_service_deployment.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial: Service creation and deployment
+This tutorial shows you how to create a simple service using one of our service templates. Additionally, you will learn how to deploy you service on the SemWES machines using our automated deployment tools.
+
+## Step 1 - Adapt service template
+
+1. Open a remote shell to your VM.
+2. Navigate to the location of the demo services (`<path/to/doc>/docs-and-training/code_examples/Python/app_simple/`).
+3. Edit the file "env", adapt the CONTEXT_ROOT, e.g, to `/my-demo`.
+
+```
+# CONTEXT_ROOT defines the deployment location relative to the host.
+CONTEXT_ROOT=/my-demo
+```
+
+## Step 2 - Deploy service using `clfpy_cli`
+
+1. Now, start `clfpy_cli` in your remote shell, and enter your user credentials and project.
+2. To select the services client, enter
+
+ ```
+ client services
+ ```
+
+3. Create a new service by choosing a short name (only lower-case letters, numbers and hyphens, e.g `my-test-service1`), and enter
```

Comment:
I would suggest a somewhat shorter name here. Also, since they will all work on the same project, name collisions should be avoided. One idea would be to use their experiment number in the name, such as `exp12-test` or similar.